src/components/Swap/Settings/TransactionTtlInput.tsx

```diff
@@ -44,20 +46,26 @@ export default function TransactionTtlInput() {
   return (
     <Column gap={0.75}>
       <Expando
-        hideRulers
-        showBottomGradient={false}
         maxHeight={4}
         open={open}
         onExpand={() => setOpen(!open)}
-        iconPrefix={<TtlValue>{ttlValue ?? placeholder}m</TtlValue>}
+        styledWrapper={false}
         title={
-          <Label
-            name={<Trans>Transaction deadline</Trans>}
-            // TODO (tina): clicking on this tooltip on mobile shouldn't open/close expando
-            tooltip={
-              <Trans>Your transaction will revert if it has been pending for longer than this period of time.</Trans>
-            }
-          />
+          <Row style={{ cursor: 'pointer' }} onClick={() => setOpen(!open)}>
```

Comment:
I thiiiiink the title already handles click-to-close? Maybe we're losing that w/ your refactor and this is adding it back?


Also, useCallback for toggle would be nice here also since you're using the anon fn implementation twice

Reply:
>  losing that w/ your refactor

that's right, this component is replacing the default title bar